packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -717,6 +717,9 @@ describe('Collection aggregations tab', function () {
   // TODO: test Collapse/Expand all stages button (currently broken)
 });
 
+// TODO: Move this test back to a single describe block to avoid a massive
+// duplication of the setup code
+//
```

Comment:
I'll do this after https://github.com/mongodb-js/compass/pull/3122 is merged to avoid conflicts between the two